apps/web/src/modules/dao/components/About/mdRender.css.ts

```diff
@@ -0,0 +1,120 @@
+import { globalStyle, style } from '@vanilla-extract/css'
+
+export const daoDescription = style({
+  wordBreak: 'break-word',
+})
+
+const mqMobile = '@media (max-width: 768px)'
+
+globalStyle(`${daoDescription} :is(h1)`, {
+  fontSize: '2.6rem',
+  fontWeight: 700,
```

Comment:
nit: fontWeight -- can be atoms

Reply:
Would be more than happy to use Atoms, except there's a couple reasons why this isn't working with globalStyle. 

1. globalStyle API doesn't seem to accept style compositions like style() does, which makes it hard to add in styles created from atoms. 

Here's what I've tried: 
![image](https://github.com/ourzora/nouns-builder/assets/53406838/f4ff91c1-0220-4696-a9ee-d96e79f48f94)
![image](https://github.com/ourzora/nouns-builder/assets/53406838/70a7eb50-771c-4466-95e4-18f257ecb755)
![image](https://github.com/ourzora/nouns-builder/assets/53406838/2d44d25a-cec4-4641-9015-26065a5886aa)

I've also searched the app to see if there's a place where we've added atoms to globalStyles. And indeed there is. However, these can only be applied globally to all elements. 

2. Extending the base style breaks the styling for the element in question. 
I tried extending the base style and adding it to individual elements ex. 
``globalStyle([${daoDescriptionH1} :is(h1)`, ``

But using a separate global style seems to break the global styles being applied. 
Before: 
![image](https://github.com/ourzora/nouns-builder/assets/53406838/a9fac458-82ac-4ff1-82db-d9fbfc2301b4)

After:
![image](https://github.com/ourzora/nouns-builder/assets/53406838/c3a2d2ba-596f-4358-a29c-31429332798a)

3. Another solution was to use complex selectors and target the individual elements from the base style. 

https://vanilla-extract.style/documentation/styling#complex-selectors

But unfortunately, globalStyles also does not support complex selectors

https://vanilla-extract.style/documentation/global-api/global-style/

4. If this were an element I had direct access to, I could apply those styles directly to className prop. But with the MD Rendered I have limited access to each element. The renders does have a way to customize some components, but I'm thinking that this would slightly convoluted given the potential gains we would receive. Especially given that the MD renderer that we use only has access to certain types of elements, and not others, which would make things pretty inconsistent.  

If there were a clean way to add atoms, I would be happy to do it. But in this case, I would recommend not using atoms. I think this would also explain why they weren't being applied in other cases where MD was used. 

https://github.com/ourzora/nouns-builder/blob/main/apps/web/src/modules/proposal/components/ProposalDescription/ProposalDescription.css.ts